LukeMason33/Rotten-Tomatillos
# Add header styling to single movie page

### Is this a fix or a feature?

Feature.

### What is the change/fix?

Small change that adds the header styling to the single movie page, rendering it without the search or filter bar.

### Where should the reviewer start?

Review the small snippet of JSX added on lines 37-42 of `single-movie-component.js`. Also note that the return has been wrapped in a fragement.

### How should this be tested?

In browser, visit a single movie page and ensure that the header is being rendered properly.
